soccer/gameplay/plays/testing/test_adaptive_formation_weights.py

```diff
@@ -0,0 +1,126 @@
+import main
+import play
+import behavior
+import evaluation.field
+import evaluation.passing_positioning
+import evaluation.shooting
+import evaluation.defensive_positioning
+import constants
+import robocup
+import math
+import enum
+
+
+class TestAdaptiveFormationWeights(play.Play):
```

Comment:
It would be nice if we could make this a normal functionality of the simulator or something. Pass it function pointers and it will display their values over the entire field. I can't think of a good place to put it though. 

If we did that, all of the commented evaluation functions could be their own separate play and it would be really simple to implement.